arch/cortex-m/src/mpu.rs

```diff
@@ -129,7 +129,7 @@ const MPU_BASE_ADDRESS: StaticRef<MpuRegisters> =
 ///
 /// There should only be one instantiation of this object as it represents
 /// real hardware.
-pub struct MPU<const NUM_REGIONS: usize> {
+pub struct MPU<const NUM_REGIONS: usize, const MIN_SIZE: u32> {
```

Comment:
Why have `MIN_SIZE` be in number of mask bits rather than just bytes?

Reply:
I thought of placing the actual value first, than I realized that this might be wrongly used. As such, I though using the bits would reduce the error space. I can change it back to use the size.

Reply:
I updated the constant to represent the size of the region